README.md

```diff
@@ -181,8 +181,8 @@ by typing:
 
 ```shell
 $ balena fleets
-ID      FLEET NAME         DEVICE TYPE      ONLINE DEVICES DEVICE COUNT
-1234567 cliFleet           fincm3           0              0
+ID NAME     SLUG           DEVICE TYPE   ONLINE DEVICES DEVICE COUNT
+1  cliFleet admin/clifleet fincm3        0              0
```

Comment:
```suggestion
1234567  cliFleet admin/clifleet fincm3        0              0
```
The ID is not shown on creation but it's shown in the table.

Reply:
applied in the commit "apply review findings" below.